11Lesson/01Homework/css/style.css

```diff
@@ -0,0 +1,269 @@
+body {
+  width: 100%;
+  font-size: 13px;
+  line-height: 24px;
+  font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  color: #666;
+  background-color: inherit;
+  margin: 0 auto;
+  background-size: cover;
+}
+
+.hero {
+  background: linear-gradient( rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3) ), url("../img/wine14.jpg");
+  color: inherit;
+  background-color: inherit;
+  height: 700px;
+  /* Create the parallax scrolling effect */
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.navbar-default {
+  background: none;
+  border: none;
+  border-bottom: 1px solid #ccc;
+}
+
+.navbar-default .navbar-nav > li > a {
+  color: #fff;
+  background-color: inherit;
+}
+
+.navbar-default .navbar-nav > li > a:hover {
+  color: #fff;
+  background-color: inherit;
+}
+
+i {
+  color: #fff;
+  background-color: inherit;
+  padding: 8px;
+}
+
+.white-text {
+  color: #fff;
+  background-color: inherit;
+}
+
+.signature {
+  max-height: 80px;
+}
+
+.left-box {
+  height: 700px;
+  font-size: 16px;
+  font-family: 'Georgia';
+  font-style: italic;
+  display: flex;
```

Comment:
Don't mix flexbox with the bootstrap grid system. If you want to use flexbox, create another container and place it **inside** of the column element that has the `col-*-*` class